distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -16,6 +16,10 @@ PROPERTIES_FILE=$HOME_DIR/etc/custom.system.properties
 RESTART_FILE="$SCRIPTDIR/restart.jvm"
 GET=${SCRIPTDIR}/get_property
 
+# Defines java garbage collection logging options
+EXTRA_JAVA_OPTS="-Xlog:gc*=info:file=${HOME_DIR}/data/log/gc.log:time,uptime,level,tags:filecount=9,filesize=20m"
```

Comment:
❓ Given this is just for starting DDF now, should it still use `${DDF_HOME}` rather than whatever user's home directory that is starting DDF?

Reply:
In this script `HOME_DIR` is DDF_HOME
```HOME_DIR=$(cd "${SCRIPTDIR}/.."; pwd -P)```